priv/perf/apps/load_test/lib/service/faucet.ex

```diff
@@ -200,7 +200,8 @@ defmodule LoadTest.Service.Faucet do
   defp deposit(faucet_account, amount, currency, deposit_finality_margin, gas_price) do
     Logger.debug("Not enough funds in the faucet, depositing #{amount} from the root chain")
 
-    {:ok, utxo} = Deposit.deposit_from(faucet_account, amount, currency, deposit_finality_margin, gas_price)
+    options = [gas_price: gas_price, deposit_finality_margin: deposit_finality_margin]
+    {:ok, utxo} = Deposit.deposit_from(faucet_account, amount, currency, options)
```

Comment:
That's the only call to `deposit_from` we have and we do not provide `return` option. Let's get rid off `:return` in `deposit_from` definition.

Reply:
> That's the only call to deposit_from we have and we do not provide return option

Because by default utxo is returned.

> Let's get rid off :return in deposit_from definition.

For some cases we need tx hash.

Reply:
Ok. I couldn't find those cases in the code.